### ISSUE TITLE
Can't Sync Audio

### ISSUE THREAD

**Issue by Hsilamot** [JasonZigelbaum/jqbx-issues#142]
I have been trying to sync audio, but it doesn't work, it always restarts the current song, but one of my friends does sync, the main difference we notice is that he sees a Progress bar of the current song while i do not see such progress bar on the web version.

Windows10 computer
Tried on Chrome, Edge. 
Tried with only Spofity Web
Tried with Spotify from Windows Store
Tried with Spotify intalled from MSI (Microsoft Installer)

Different Computer with Chrome with no spotify installed Windows7

**Comment by JasonZigelbaum** [comment]
Can you check if the clock on your computer is set accurately? That has
caused this issue in the past. Let me know.

**Comment by Hsilamot** [comment]
yes this solved the problem, i took so long to respond since i was thinking if there was another way of syncing but the only way would be to put an individual sync for the time

**Comment by nickhans** [comment]
I have this same problem with the macOS app. When I sync audio the song restarts. My computer clock is set accurately. The current song playing also restarts after I listen to a 30 second preview of a song in my queue.

**Comment by JasonZigelbaum** [comment]
Hey sorry you're having this issue. Have you tried restarting the Spotify application on your machine?

**Comment by nickhans** [comment]
Yeah I restarted spotify and jqbx and it ended up working fine thank you!
